keyboards/frobiac/hypernano/hypernano.c

```diff
@@ -0,0 +1,11 @@
+// Copyright 2023 @frobiac
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "hypernano.h"
+
+/// hook up tp_reset() in matrix init.
+void keyboard_pre_init_user() {
+#if defined(PS2_MOUSE_ENABLE) && defined(PS2_RESET_PIN)
+    tp_reset();
+#endif
+}
```

Comment:
If this functionality is required for the board to work, the function needs to be added here, not just called from your userspace. If this isn't the case, this file should just be deleted.
```suggestion
```

Reply:
Initially I was just trying to avoid duplicating the code, but it looks as if the userspace approach is not the most straight-forward one for features like these. Added now with latest fixes.